tests/test_pubsub.py

```diff
@@ -474,3 +475,31 @@ def t_run(rc):
             t.start()
     except Exception:
         print("Error: unable to start thread")
+
+
+class TestPubSubPubSubSubcommands(object):
+
+    @skip_if_server_version_lt('2.8.0')
+    def test_pubsub_channels(self, r):
+        p = r.pubsub(ignore_subscribe_messages=True)
+        p.subscribe('foo', 'bar', 'baz', 'quux')
+        channels = sorted(r.pubsub_channels())
+        assert channels == [b('bar'), b('baz'), b('foo'), b('quux')]
+
+    @skip_if_server_version_lt('2.8.0')
+    def test_pubsub_numsub(self, r):
+        p1 = r.pubsub(ignore_subscribe_messages=True)
```

Comment:
could this not be optimized to only `r.pubsub(ignore_subscribe_messages=True).subscribe('foo', 'bar', baz')` No need for the variabel defenition then.